app/views/users/show.html.erb

```diff
@@ -26,16 +29,24 @@
           <h5 class="card-title">Post #<%= @recent_posts.index(post) + 1%></h5>
         </a>
```

Comment:
- [ ] (optional) Kindly fix the following bug. On the users post index page, you can see the 3 most recent posts as expected, however they are not being displayed according to their post title's making them confusing
![image](https://github.com/Nahid1911/blog-app-ruby-on-rails/assets/81932478/44a5beb9-dee7-475a-a04c-71afb58b47f1)

If you see the first post in the screenshot is actually the fourth post, the second post is the third post and so on, Although clicking on the post reveals the actual title, kindly fix this part as well to show the correct Post title

Actual Individual Post Page
![image](https://github.com/Nahid1911/blog-app-ruby-on-rails/assets/81932478/42120be6-4d04-4ac5-bed4-e2b08cf81b5c)

Reply:
- [ ] Since this was missed on the last milestone, you could fix it here for a better user experience